cellrank/tl/estimators/_gpcca.py

```diff
@@ -32,7 +32,7 @@
 @d.dedent
 class GPCCA(BaseEstimator, Macrostates, Schur, Eigen):
     """
-    Generalized Perron Cluster Cluster Analysis [GPCCA18]_.
+    Generalized Perron Cluster Cluster Analysis [GPCCA18]_ and implemented in `pyGPCCA <https://pygpcca.readthedocs.io/en/latest/>`_.
```

Comment:
@michalk8 mb. a typo: "...as implemented..." instead of "...and implemented..."?